diff --git a/package.json b/package.json
index 24204177b341f86369c7012da9e4bde27de5ebe3..0be7a70fd09c732ca37bab9a5596480929d6f9ca 100644
--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
   "description": "a libre collaborative resources mapper powered by open-knowledge (client-side)",
   "license": "AGPL-3.0",
   "homepage": "https://inventaire.io",
+  "version": "0.0.0",
   "repository": {
     "type": "git",
     "url": "http://github.com/inventaire/inventaire-client"
@@ -28,7 +29,7 @@
     "lint-fix": "./scripts/lint_fix",
     "lint-staged": "./scripts/lint_staged",
     "node-exec": "node --no-warnings --loader ./custom-loader.js --es-module-specifier-resolution=node",
-    "postinstall": "./scripts/postinstall",
+
     "start": "webpack serve --config ./bundle/webpack.config.dev.cjs",
     "test": "./scripts/run_unit_tests",
     "test-fast": "./scripts/run_fast_tests",
@@ -39,7 +40,7 @@
   },
   "dependencies": {
     "autosize": "^4.0.2",
-    "backbone": "^1.1.2",
+
     "backbone-filtered-collection": "^0.4.0",
     "backbone-nested": "^2.0.4",
     "backbone.marionette": "git+https://github.com/inventaire/backbone.marionette.git#custom-dependencies",
@@ -70,7 +71,8 @@
     "papaparse": "^4.6.3",
     "quagga": "^0.12.1",
     "underscore": "^1.8.3",
-    "wikidata-lang": "^2.0.11"
+    "wikidata-lang": "^2.0.11",
+    "tiny-chalk": "^2.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
@@ -87,7 +89,7 @@
     "eslint-plugin-svelte3": "^3.0.0",
     "file-loader": "^6.2.0",
     "handlebars-loader": "^1.7.1",
-    "html-webpack-plugin": "^4.5.0",
+    "html-webpack-plugin": "^5.1.0",
     "iconv-lite": "^0.4.24",
     "mini-css-extract-plugin": "^1.3.5",
     "mocha": "^8.1.3",
diff --git a/scripts/build_i18n b/scripts/build_i18n
index 4165369cec6ad1dea0f4987a6082b973c99adcf6..6d2c597ad2f4d7140119ea9cb241897b13f402d5 100755
--- a/scripts/build_i18n
+++ b/scripts/build_i18n
@@ -3,22 +3,22 @@
 # Keep 'inventaire-i18n' as the directory name to make this directory easy to ignore
 # in gitignore or text editor exclude patterns

-[ -e inventaire-i18n ] && {
-  cd inventaire-i18n
-  # Update dist files
-  git fetch origin dist
-  git checkout origin/dist
-  git checkout -B dist
-  cd ..
-} || {
-  git clone https://github.com/inventaire/inventaire-i18n.git --branch dist
-}
+# [ -e inventaire-i18n ] && {
+#   cd inventaire-i18n
+#   # Update dist files
+#   git fetch origin dist
+#   git checkout origin/dist
+#   git checkout -B dist
+#   cd ..
+# } || {
+#   git clone https://github.com/inventaire/inventaire-i18n.git --branch dist
+# }

-mkdir -p ./public/i18n
-echo "copying dist files"
-cp inventaire-i18n/dist/client/* ./public/i18n
+# mkdir -p ./public/i18n
+# echo "copying dist files"
+# cp inventaire-i18n/dist/client/* ./public/i18n
+# echo "copying languages_data.js to app/lib"
+# cp inventaire-i18n/dist/languages_data.js ./app/lib

-echo "copying languages_data.js to app/lib"
-cp inventaire-i18n/dist/languages_data.js ./app/lib
 sed -i 's/module.exports =/export default/' ./app/lib/languages_data.js
 ./node_modules/.bin/eslint --fix ./app/lib/languages_data.js
diff --git a/scripts/build b/scripts/build
index 27ebea33b386effe49d7fcd06671b26916880ab8..dbb44fe438abea2c7f1660c9ee2b47e0e28d0e55 100755
--- a/scripts/build
+++ b/scripts/build
@@ -3,10 +3,10 @@
 # Exit on error (cf http://jvns.ca/blog/2017/03/26/bash-quirks/)
 set -e

-echo -e '\e[0;30mupdating i18n\e[0m'
-./scripts/build_i18n
-
+# echo -e '\e[0;30mupdating i18n\e[0m'
+# ./scripts/build_i18n
 echo -e '\e[0;30mupdate mentions\e[0m'
+node --version
 npm run update-mentions

 echo -e '\e[0;30mwebpack build\e[0m'
@@ -14,7 +14,7 @@ rm -f ./public/dist/*
 webpack --config ./bundle/webpack.config.prod.cjs --progress

 utc_time=$(date -Ins | sed 's/,.*//')
-last_commit_hash=$(git rev-parse --short HEAD)
+last_commit_hash=$(git rev-parse --short HEAD || echo "XXXXXXXX")
 bundle_name="${utc_time}_commit_${last_commit_hash}"
 mkdir -p ./scripts/assets/bundles_archives ./scripts/assets/bundles_stats
 archive_path="./scripts/assets/bundles_archives/${bundle_name}"
@@ -47,13 +47,13 @@ gzip -9kf ./public/dist/*
 if [ ! -d "./public/sitemaps" ] ; then
   echo -e '\e[0;30mgenerate sitemaps\e[0m'
   mkdir ./public/sitemaps
-  npm run generate-sitemaps
+  # npm run generate-sitemaps
 fi

 echo -e '\e[0;30mgzip assets\e[0m'
 gzip -9kf public/fonts/*
 gzip -9kf public/json/*
-gzip -9kf public/sitemaps/*
+# gzip -9kf public/sitemaps/*
 gzip -9kf public/*.xml
 gzip -9kf public/*.jsonld
 gzip -9kf public/*.txt
